client/src/views/header/NotificationCenterPopover.vue

```diff
@@ -52,6 +61,10 @@ onMounted(() => {
   notificationManager.addToList(testNotification);
 });
 
+function clearReadNotifications(): void {
+  notificationManager.getOnlyUnReadNotification();
```

Comment:
I see why you did this, hmm I'm not sure we want it this way, we should probably just hide them when they are read, not remove them completely from the list @Max-7?